susemanager/src/mgr_bootstrap_data.py

```diff
@@ -435,7 +435,7 @@
     "python3-spacewalk-check",
     "python3-spacewalk-client-setup",
     "python3-spacewalk-client-tools",
-    "python3-uyuni-common-libs",
+    "python3-uyuni-common-libs*",
```

Comment:
Maybe document somewhere that the `*` means it's optional?
This might confuse people in the future.

Reply:
I added a format comment on top of this file

Reply:
Thank you!
Feel free to merge the PR :smile: